collisions_main.py

```diff
@@ -0,0 +1,246 @@
+"""This is the main module for an elastic collision physics simulation
+of circular objects.
+"""
+
+import decimal
+import math
+import sys
+from random import randrange
+
+import pygame as pyg
+import pygame.freetype
+
+import collisions_functions as funcs
+
+
+pygame.init()
+
+# Base simulation parameters
+XDIM = 1300
+YDIM = 1000
+win = pygame.display.set_mode((XDIM, YDIM))
+pygame.display.set_caption("Collisions")
+
+TIMESTEP = 10
+red = (255, 0, 0)
+green = (0, 255, 0)
+blue = (0, 0, 255)
+black = (0, 0, 0)
+boundary = [100, 900, 100, 900]
+BOUNDARY_WIDTH = 1
+max_vel = 1
+max_rad = 25
+atom_counter = 0
+AREA_CAPACITY = 0.20
+capacity = 0
+
+# Collision section parameters
+base_section_dim = max_rad * 5
+overlap_dim = max_rad
+num_x_sections = int(((boundary[1] - boundary[0]) / base_section_dim) + 1)
+num_y_sections = int(((boundary[3] - boundary[2]) / base_section_dim) + 1)
+
+# Slider control variables
+past_button_state = 0
+BASE_SLIDER_Y = 110
+click_x = 0
+click_y = 0
+OBJ_SLIDER_X_MIN = 955
+OBJ_SLIDER_X_MAX = 1045
+OBJ_SLIDER_MULT = 1
+RAD_SLIDER_X_MIN = 1055
+RAD_SLIDER_X_MAX = 1145
+RAD_SLIDER_MULT = 0.2
+VEL_SLIDER_X_MIN = 1155
+VEL_SLIDER_X_MAX = 1245
+VEL_SLIDER_MULT = 0.01
+
+font = pygame.freetype.SysFont(pygame.font.get_default_font(), 18)
+
+atoms = []
+
+# Manually set initial conditions for testing purposes
+"""
+atoms.append(funcs.Atom(50, red, 0, boundary))
+atoms[0].ID = 0
+atoms[0].mass = atoms[0].rad**2 * math.pi
+atoms[0].vel = [-2, 0]
+atoms[0].x = 650
+atoms[0].y = 450
+atom_counter += 1
+
+atoms.append(funcs.Atom(50, blue, 0, boundary))
+atoms[1].ID = 1
+atoms[1].mass = atoms[1].rad**2 * math.pi
+atoms[0].vel = [2, 0]
+atoms[1].x = 350
+atoms[1].y = 530
+atom_counter += 1
+"""
+
+
+# Simulation loop
+run = True
+while run:
+    # Creates time delay of TIMESTEP in ms
+    pygame.time.delay(TIMESTEP)
+    
+    # Check for ESC key pressed or pygame window closed, to quit
+    for event in pygame.event.get():
+        if event.type == pyg.QUIT:
+            pygame.quit()
+            sys.exit()
+        elif event.type == pyg.KEYDOWN:
+            if event.key == pyg.K_ESCAPE:
+                pygame.quit()
+                sys.exit()
+
+    # Refresh screen and draw boundary
+    win.fill((255, 255, 255))
+    pygame.draw.rect(win, red, pygame.Rect(100, 100, 800, 800), BOUNDARY_WIDTH)
+
+
+    # Object number slider
+    if click_x >= OBJ_SLIDER_X_MIN and click_x <= OBJ_SLIDER_X_MAX:

```

Comment:
I think python has some neat syntax which allows you to express this as

`OBJ_SLIDER_X_MIN <= click_x <= OBJ_SLIDER_X_MAX`